assets/style.css

```diff
@@ -83,7 +131,255 @@ body {
+        color: #172B4D;
+}
+
+#recent_works .indicator {
+       margin-top: 24px;
+       margin-bottom: 62px;
+       background-color: #36B37E;
+       border-radius: 24px;
+       width: 48px;
+       height: 4px;
+}
+
+.work_row {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        padding: 4px 2px;
+        gap: 24px;
+        width: 331px;
+        height: 2972px;
+        flex: none;
+        order: 1;
+        flex-grow: 0;
+}
+
+.card {
+        transition: 0.3s;
+        width: 327px;
+        border: 1px solid #D0D9D4;
+        border-radius: 8px;
+}
+
+.card img {
+        width: 100%;
+        height: 252px;
+}
+
+.card-footer {
+        display: flex;
+        flex-direction: column;
+}
+
+.card .card-footer .tags {
+        display: flex;
+        flex-wrap: wrap;
+        flex-direction: row wrap;
+        justify-content: center;
+        align-items: center;
+        list-style-type: none;
+        gap: 8px;
+        width: 266px;
+}
+
+.project_title {
+        width: 295px;
+        height: 57px;
+        font-family: 'Inter', sans-serif;
+        font-style: normal;
+        font-weight: 600;
+        font-size: 24px;
+        line-height: 32px;
+        display: flex;
+        align-items: center;
+        text-align: center;
+        color: #3A4A42;
+        flex: none;
+        order: 0;
+        flex-grow: 0;
+}
+
+.action {
+        width: 305px;
+        height: 69px;
+        flex: none;
+        order: 2;
+        flex-grow: 0;
+}
+
+.action button {
+        width: 122px;
+        height: 48px;
+        bottom: 8px;
+        font-weight: 500;
+        font-size: 17px;
+        line-height: 24px;
+        letter-spacing: 0.03em;
+}
+
+#about_me {
+       background-image: url('./images/illustration-about-me.png');
+       background-repeat: no-repeat;
+       background-position: center;
+       background-size: 100%;
+       margin-top: 455px;
+       width: 100%;
+       height: 389.38px;
+}
+
+.supporting_text {
+        height: 189px;
+        font-weight: 400;
+        font-size: 20px;
+        line-height: 28px;
+        text-align: center;
+}
+
+.contact_form_text {
+        font-weight: 700;
+        font-size: 40px;
+        line-height: 52px;
+        text-align: center;
+}
+
+.btn-resume {
+        padding: 12px;
+        gap: 10px;
+        width: 159px;
+        height: 48px;
+        background: #36B37F;
+        border-radius: 4px;
+        font-weight: 500;
+        font-size: 17px;
+        line-height: 24px;
+        flex: none;
+        order: 2;
+        flex-grow: 0;
+}
+
+#me_knowledge {
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+        justify-content: center;
+        align-items: center;
+}
+
+.works_card {
+        transition: 0.3s;
+        width: 327px;
+        height: 352px;
+        background-color: #EBF0EE;
+        border: 1px solid #D0D9D4;
+        border-radius: 8px;
+        margin-top: 24px;
+}
```

Comment:
Kindly ensure that your design is as close to the original design as much as possible. Check and rectify this disparity in the design between yours and the figma below. 

Your design

![Screenshot from 2022-09-23 16-58-23](https://user-images.githubusercontent.com/65233860/192013408-fb5d825d-3c78-4efc-b429-3e3dd7eda664.png)

Figma design
![Screenshot from 2022-09-23 16-58-46](https://user-images.githubusercontent.com/65233860/192013430-ff5a8818-50d1-4ccf-afa7-c8fe07630b4e.png)